src/renderer/services/file-management-system/index.ts

```diff
@@ -427,23 +429,30 @@ export default class FileManagementSystem {
           throw new Error("FileId was not published on COMPLETE upload: " + fssStatus.uploadId)
```

Comment:
hmm I'm not sure I get what you are saying. IMO this is a "resume" scenario because while FSS has completed its portion that isn't the whole story and the FUA has to "resume" the upload to complete its portion (line 431).

Reply:
Ah, I get it.  FUA would still need to attach metadata, right?

Reply:
Yep! & clean up its JSS job/mark it as complete